README.md

```diff
@@ -1,2 +1,41 @@
-# py-port-scan
-Python module for quick port scanning.
+# Port Scanner
+
+Port Scanner is a python library to scan the mentioned ports of the given IP address.
+
+## Installation
+
+Use the package manager [pip](https://pip.pypa.io/en/stable/) to install the Port Scanner.
+
+```bash
+pip install PortScanner
+```
+
+## Usage
+
+```python
+import PortScanner
+
+mulScan = PortScanner.MultiScan(
+        targets=["List of target IPs"],
+        ports=["List of ports to scan"],
+        threads=100, # Number of threads in the thread pool
+        timeout=1, # Seconds to socket timeout
+        proxy_ip=["List of Proxy IPs"],
+        proxy_port=["List of proxy ports"]
+)
+
+# To Scan all the ports ie 0-65535 for the given targets
+mulScan.run_full_scan()
```

Comment:
This method does not return anything, Essentially I believe that it does tell you it scanned, but the results are not collected from threads and returned to the user! I suppose it will defeat the use case scenario.

Reply:
Updated, now the function returns a dictionary having all the results.